doc/lua_api.md

```diff
@@ -5714,7 +5714,7 @@ Global callback registration functions
 Call these functions only at load time!
 
 * `minetest.register_globalstep(function(dtime))`
-    * Called every server step, usually interval of 0.1s
+    * Called every server step, usually interval of 0.1s, also dtime is measured in seconds.
```

Comment:
This explanation is not explaining *what* is `dtime`. I would suggest the following:

```markdown
* Called every server step, usually every 0.1s
* `dtime` is the time since last execution in seconds
```